src/python/ripe_demo/static/js/ripe.js

```diff
@@ -1397,6 +1397,30 @@ if (typeof require !== "undefined") {
     var ripe = base.ripe;
 }
 
+ripe.Ripe.prototype.getLocale = function(values, locale, options, callback) {

```

Comment:
the name of this method should be `localeMultiple` to be in line with the name locale_m on the server side